src/modules/Profile.rb

```diff
@@ -13,26 +13,10 @@ class ProfileClass < Module
     # All these sections are handled by AutoYaST (or Installer) itself,
     # it doesn't use any external AutoYaST client for them
     GENERIC_PROFILE_SECTIONS = [
-      # Booting is configured during the installation
-      "bootloader",
-      # Bug: this is already in every auto-generated profile
-      "deploy_image",
```

Comment:
deploy_image might but might not be on the system, but every generated profile contains it. But if it contains use=false, then it will report an error if I'm not mistaken